Website/src/rendering/src/components/HeroSection.tsx

```diff
@@ -1,50 +1,40 @@
-import { ComponentProps } from 'lib/component-props';
-import { Text, Field, LinkField, Link, ImageField, Image } from '@sitecore-jss/sitecore-jss-nextjs';
+import { Text, Field, RichText, ImageField } from '@sitecore-jss/sitecore-jss-nextjs';
+import { ComponentWithChildrenProps } from 'lib/component-props';
+import { Placeholder } from '@sitecore-jss/sitecore-jss-nextjs';
```

Comment:
This should be merged with the first import line from the same NPM package.

Reply:
Updated